docs/src/main/sphinx/security/cert-auth.rst

```diff
@@ -0,0 +1,108 @@
+==========================
+Certificate authentication
+==========================
+
+Trino can be configured to support certified clients seeking validation from a
+Trino server. This authentication method is only provided to support sites that
+have an absolute requirement for client authentication *and already have* client
+certificates for each client. Sites in this category have an existing PKI
+infrastructure, possibly including an onsite CA.
+
+This feature is *not* appropriate for sites that would need to generate a set of
+client certificates in order to use this authentication method. Starburst
+recommends instead configuring one of the password-based authentication methods,
```

Comment:
This sentence reads awkwardly to me, would a list work better since its a grouping of 3 alternatives?

Reply:
+1. Also, it's better to emphasis what they should do rather than give a visual bump to a no-no.